File: Python/1183.py
operacao = input()
matriz = [[0.0]] * 12
total = 0.0
for i in range(12):
    linha = [0.0] * 12
    for j in range(12):
        linha[j] = float(input())
    matriz[i] = linha
for i in range(1, 12):
    for j in range(0, i):
        total += matriz[j][i]
if operacao == 'S':
    print("%.1f" % total)
else:
    print("%.1f" % (total / 66))

File: C/1183.c
#include <stdio.h>

int main() {

    char operacao;
    double matriz[12][12];
    double total;
    int i, j, k = 1;
    scanf("%c", &operacao);

    for (i = 0; i < 12; i++) {
        for (j = 0; j < 12; j++) {
            scanf("%lf", &matriz[i][j]);
        }
    }

    for (i = 0; i < 12; i++) {
        for (j = k; j < 12; j++) {
            total += matriz[i][j];
        }
        k++;
    }

    if (operacao == 'S') {
        printf("%.1lf\n", total);
    } else {
        printf("%.1lf\n", (total / 66.0));
    }

    return 0;

}
